examples/rabbitmq-simple-producer/rabbitmq_simple_producer.bal

```diff
@@ -16,7 +16,7 @@ public function main() {
         io:println("The queue was created successfully");
     }
 
-    // Publishes the message using the routing key named "testing".
+    // Publishes the message using the routing key named "testQueue".
```

Comment:
Shall we use a proper name for these queues instead of test prefix?

Reply:
Fixed